docs/snippets/providers/mailgun-snippet-autogenerated.mdx

```diff
@@ -0,0 +1,17 @@
+{/* This snippet is automatically generated using scripts/docs_render_provider_snippets.py 
+Do not edit it manually, as it will be overwritten */}
+
+## Authentication
+This provider requires authentication.
+- **email**: Email address to send alerts to (required: False, sensitive: False)
+- **sender**: Sender email address to validate (required: False, sensitive: False)
+- **extraction**: Extraction Rules (required: False, sensitive: False)
+
+
+## In workflows
+
+This provider can't be used as a "step" or "action" in workflows. If you want to use it, please let us know by creating an issue in the [GitHub repository](https://github.com/keephq/keep/issues).
+
+
+
+If you need workflow examples with this provider, please raise a [GitHub issue](https://github.com/keephq/keep/issues).
```

Comment:
it's a duplicate, no?

Reply:
One is about actions and steps, the other is about examples...

Reply:
still doesn't make sense for me - you can't ask for workflow example if it doesn't have steps or actions, no?

Reply:
Got it :)